rules_conditions.md

```diff
@@ -0,0 +1,226 @@
+# Refinery Conditions
+
+## Overview
+
+Refinery rules are described as a series of conditions.
+Each condition is composed from a combination of these named parameters:
+
+- `Field` (or `Fields`)
+- `Operator`
+- `Value`
+- `Datatype`
+
+The `Operator` is never optional, and controls which of the parameters are required and which are optional.
+
+## `Field`
+
+The `Field` parameter points to a specific named element in the trace data.
+If a `Field` is named within a span, then it `exists`.
+A specific `Field` may or may not exist on any specific span in a trace.
+It might not even exist within a trace at all.
+
+When a `Field` is absent in all spans within a trace, the associated rule does not apply to that trace.
+
+A `Field` is always a single string
+A `Field` is always matched by exact comparison.
+No transformations for case or punctuation are performed.
+
+### Example use of `Field`
+
+```yaml
+Conditions:
+    Field: http.route
+    Operator: =
+    Value: /health-check
+```
+### Leveraging Special Refinery Telemetry in Root Spans
+
+Some Refinery configuration options introduce special names that are added to telemetry.
+
+For example, when `AddCountsToRoot` is enabled, `meta.span_count` is added to all root spans, allowing for the creation of rule conditions based on span counts.
+
+```yaml
+Conditions:
+    Field: "meta.span_count"
+    Operator: ">"
+    Value: 300
+    Datatype: int
+```
+
+In this scenario, the rule applies to traces with more than 300 spans.
```

Comment:
I'd move this to line 41, so users understand the context of the yaml example that they're about to view.
```suggestion
In this `meta.span_count` example, the Refinery rule applies to traces with more than 300 spans.
```